### PR TITLE
a drop of documentation

### PR DESCRIPTION
**Pull request by ghost**
### What was wrong?

see #5 

### How was it fixed?

Added a text file for more details

#### Cute Animal Picture

dolphin

### CODE REVIEW

**Comment by pipermerriam** [comment]
I'd do it myself but there isn't an easy way to remove a file using the web-based-editor (that I know of).  Seems appropriate to just inline this instead of creating a new document.  Mind making the change?

File: what_Is_eth-bloom.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+to help data stored within them to be retrieved or stored.  They work like other
+data structures, but in a probabilistic way: it allows for false positive
+matches but not false negative matches.  Bloom filters storage space use is
+low relative to other kinds of data structures.
+
+( For more information on Bloom filters, see Wikipedia: https://en.wikipedia.org/wiki/Bloom_filter )
+
+A trie is an a kind of tree data structure for use with dynamic data
+
+Ethereum bloom filters are bloom filters implemented using a "trie" data
```
Comment (pipermerriam) [inline what_Is_eth-bloom.txt:13]: I don't believe this is entirely accurate.  The bloom filter has nothing to do with the trie data other than the fact that some of the data that the bloom filter holds *happens* to also be stored in the ethereum state.

I am of the opinion that mentioning trie structures is only confusing in this context.

Comment (ghost) [reply what_Is_eth-bloom.txt:13]: that makes a lot more sense.

Comment (ghost) [reply what_Is_eth-bloom.txt:13]: (and will make said change)

**Pull request closed without merging**